### PR TITLE
Feature/13729 UI automation sprint1 pb is

### PR DESCRIPTION
**Pull request by kishorambare**
Hi All,

This PR is consist of UI automation of sprint 1 PBI.
Kindly review and approve it.

Thanks.

### CODE REVIEW

File: Tests/FunctionalTests/ess-ui-home.spec.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        await LoginPortal(page,autoTestConfig.user, autoTestConfig.password, commonObjectsConfig.loginSignInLinkSelector);
+        await page.locator(fssHomePageObjectsConfig.essLinkSelector).click();
+        await page.waitForTimeout(500);
+        await expect(page.url()).toContain("/exchangesets");
+        await expect(page).toHaveTitle("Admiralty - File Share Service - Exchange Sets");        
+    });
+
+    test('Verify appending "/search" in  url after login, navigates to Search page', async ({ page }) => {
+        await LoginPortal(page,autoTestConfig.user, autoTestConfig.password, commonObjectsConfig.loginSignInLinkSelector);
+        await page.goto(autoTestConfig.url+"/#"+"/search");
```
Comment (crdandoukho) [inline Tests/FunctionalTests/ess-ui-home.spec.ts:46]: Please simplify
```suggestion
        await page.goto(autoTestConfig.url+"/#/search");
```

File: Tests/FunctionalTests/ess-ui-home.spec.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        await page.waitForLoadState('load');
+        expect(await page.innerHTML(fssSearchPageObjectsConfig.searchPageContainerHeaderSelector)).toEqual(fssSearchPageObjectsConfig.searchPageContainerHeaderText); 
+      
+    });
+
+    test('Verify appending "/exchangesets" in  url after login, navigates to ESS laanding page', async ({ page }) => {
+        await LoginPortal(page,autoTestConfig.user, autoTestConfig.password, commonObjectsConfig.loginSignInLinkSelector);
+        await page.goto(autoTestConfig.url+"/#"+"/exchangesets");
```
Comment (crdandoukho) [inline Tests/FunctionalTests/ess-ui-home.spec.ts:54]: Please simplify
```suggestion
        await page.goto(autoTestConfig.url+"/#/exchangesets");
```

File: Tests/FunctionalTests/ess-ui-home.spec.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+    test('Verify clicking Search link from home page after login, navigates to Search page', async ({ page }) => {
+        await LoginPortal(page,autoTestConfig.user, autoTestConfig.password, commonObjectsConfig.loginSignInLinkSelector);
+        await page.locator(fssHomePageObjectsConfig.searchLinkSelector).click();
+        await page.waitForLoadState('load');
+        await expect(page).toHaveTitle("Admiralty - File Share Service - Search");
+        expect(await page.innerHTML(fssSearchPageObjectsConfig.searchPageContainerHeaderSelector)).toEqual(fssSearchPageObjectsConfig.searchPageContainerHeaderText);      
+    });
+
+    test('Verify clicking Exchange sets link after login navigates to ESS laanding page', async ({ page }) => {
```
Comment (crdandoukho) [inline Tests/FunctionalTests/ess-ui-home.spec.ts:36]: Typo
```suggestion
    test('Verify clicking Exchange sets link after login navigates to ESS landing page', async ({ page }) => {
```

**Review by crdandoukho — CHANGES_REQUESTED** [review]
Minor changes needed please.

I notice the tests are failing because the /exchangesets page title is not being set correctly. The app should be fixed to set the title correctly, the tests themselves look good to me and should not be changed

File: Tests/FunctionalTests/ess-ui-home.spec.ts (hunk 1 above)
Comment (kishorambare) [reply Tests/FunctionalTests/ess-ui-home.spec.ts]: HI Chris,
Resolved the review comments.
Pls have a look.

File: Tests/FunctionalTests/ess-ui-home.spec.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+    test('Verify appending "/search" in  url after login, navigates to Search page', async ({ page }) => {
+        await LoginPortal(page,autoTestConfig.user, autoTestConfig.password, commonObjectsConfig.loginSignInLinkSelector);
+        await page.goto(autoTestConfig.url+"/#/search");
+        await page.waitForLoadState('load');
+        expect(await page.innerHTML(fssSearchPageObjectsConfig.searchPageContainerHeaderSelector)).toEqual(fssSearchPageObjectsConfig.searchPageContainerHeaderText); 
+      
+    });
+
+    test('Verify appending "/exchangesets" in  url after login, navigates to ESS laanding page', async ({ page }) => {
```
Comment (RaviK2233) [inline Tests/FunctionalTests/ess-ui-home.spec.ts:52]: typo error for `landing` .

**Review by RaviK2233 — CHANGES_REQUESTED** [review]

File: Tests/FunctionalTests/ess-ui-home.spec.ts (hunk 4 above)
Comment (kishorambare) [reply Tests/FunctionalTests/ess-ui-home.spec.ts:52]: Done.
Thanks Ravi.

**Review by RaviK2233 — APPROVED** [review]

**Review by crdandoukho — APPROVED** [review]
Looks good

**Pull request merged**